packages/mobile/src/screens/app-screen/AppTabScreen.tsx

```diff
@@ -273,14 +265,7 @@ export const AppTabScreen = ({ baseScreen, Stack }: AppTabScreenProps) => {
         />
         <Stack.Screen name='ChangeEmail' component={ChangeEmailModalScreen} />
       </Stack.Group>
-      <Stack.Group>
```

Comment:
Interesting so we are going away from the stack group? What's main rationale here? State management?